jsk_hand/softhand_ros/euslisp/softhand-v2-interface.l

```diff
@@ -0,0 +1,190 @@
+(ros::roseus-add-msgs "std_msgs")
+(ros::roseus-add-msgs "dynamixel_controllers")
+(ros::roseus-add-srvs "dynamixel_controllers")
+
+
+(defclass softhand-v2-interface
+  :super propertied-object
+  :slots (finger1-controller-name
+          finger2-controller-name
+          finger3-controller-name
+          finger1-command-topic-name
+          finger2-command-topic-name
+          finger3-command-topic-name
+          finger1-calib-action-client
+          finger2-calib-action-client
+          finger3-calib-action-client
+          finger1-set-torque-limit-service-name
+          finger2-set-torque-limit-service-name
+          finger3-set-torque-limit-service-name
+          heater-controller-name
+          heater1-command-topic-name
+          heater2-command-topic-name
+          heater3-command-topic-name
+          thumb-controller-name
+          thumb-command-topic-name
+  ))
+
+(defmethod softhand-v2-interface
+  (:init (&key (calibrate nil) (namespace nil))
+    (ros::rate 10)
+    (setq finger1-controller-name
```

Comment:
If you're planning to develp v3/v4 hand please consider set variables programmably. If v1 is almost absolate, we do not change v1 code. Our ambition is to create `hand-interface.l` in future, so that @Utaro-M can use this too.
```
16.irteusgl$ (dotimes (i 3) (set (intern (format nil "FINGER~A-VAR" i)) (* i 10)))
nil
17.irteusgl$ (dotimes (i 3) (print (eval (intern (format nil "FINGER~A-VAR" i)))))
0
10
20
```
you should not write too complex code, but 
```
(setq finger-controller-names nil)
(dotimes (i 3) (push (set (intern (format nil "FINGER~A-CONTROLLER-NAME" i)) (format nil "finger~A_joint_controller" i)) finger-controller-names))
(dolist (name finger-controller-names)
    ....
(:calibrate-finger-impl (id) ...)
(:calibrate-finger1 () (send self :calibrate-finger-impl 1) ;; you can genrate this method programmably, but may be it too complex?
(:finger-rotate-impl (id) .. )
(:finger-rotate0 () (send self :finger-rotate-impl 0))
(:finger-rotate1 () (send self :finger-rotate-impl 1))
;;;
;;; hardare specific method
(:thumb-rotate (&rest args) (send* self :finger-rotate0 args));
.....
```

Reply:
I update to make `dynamixel-hand-inteface.l` and merge codes.